test/checks/shared/avoid-inline-spacing.js

```diff
@@ -0,0 +1,83 @@
+describe('avoid-inline-spacing tests', function() {
```

Comment:
Should test when an inline style has an invalid value, like `line-height: 3foobar !important;`